docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+
+- Login options
+- Discovering 'My Collections'
+- Creating a **private** collection
+- Adding datasets to a newly created collection
+- Sharing **private** collections with collaborators via private links
+- Publishing your collection for public access
+
+### Portal Sign in
+
+sign in options for the portal (gmail, github, etc)
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113528770-9f46a080-958f-11eb-83a1-36620e1543d2.png) --->
+
+<p align="center">
+  <img width="250" src="https://user-images.githubusercontent.com/25663501/113528770-9f46a080-958f-11eb-83a1-36620e1543d2.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Login options
+</p>
+
+<br/>
+
+The cellxgene data portal offers multiple login options including sign-in with your gsuite and github accounts, or via email address.
+
+<br/>
+
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113530303-a96a9e00-9593-11eb-91b7-0898d521314a.png) --->
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/113530303-a96a9e00-9593-11eb-91b7-0898d521314a.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Navigate to "My Collections"
+</p>
+
+<br/>
+
+It should be noted that collections are displayed with specific metadata about the collections (including assays(s) used, organism(s) profiled, number of cells in the collection, the tissue profiled, and the disease state studied). After you have logged in,  you will be able to navigate to the 'My Collections' page where you can view all of the datasets that you have uploaded to the portal (if you are just starting out, the 'My Collections' view will be empty).
+
+
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113528943-18de8e80-9590-11eb-92ab-2a606879b70c.png) --->
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113530607-6d840880-9594-11eb-8af7-17f75d3a8a40.png) --->
+
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/113530607-6d840880-9594-11eb-8af7-17f75d3a8a40.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> My Collections
+</p>
+
+<br/>
+
+In the image above, we see an example of how the 'My Collections' page will look after you have uploaded a few datasets to the dataportal. Note that some of the collections in this example are published (publicly available) and one is private. The private collection is only available to people who you share the private link with. This allows for control over who views your datasets and for making amendmendents to datasets that have been uploaded to the portal previously. To add a new collection, we can simply click on the button highlighted above. 
```

Comment:
_dataportal -> data portal_

_amendmendents -> amendments_

We use the term "revisions". 

_This allows for control over who views your datasets and for making amendmendents to datasets that have been uploaded to the portal previously._ - until **private revisions of public collections** is available, the curator must delete the dataset and upload it again to revise.

Reply:
I'm not too keen on documenting future functionality. I think if it is available to curators, then we should mention it, but if it is not live, then it should not be included in the tutorial

Reply:
keep a section of future functionality to let users know that it is on the roadmap (in cases where users are expecting it)